eng/common/scripts/Analyze-Changes.ps1

```diff
@@ -138,6 +142,15 @@ function Get-ActiveVariables($changes) {
         $variables += "RunCore"
     }
 
+    foreach ($rule in $pathBasedRules.Keys) {
+        foreach ($rulePath in $pathBasedRules[$rule]) {
+            if ($runAll -or $root.PathExists($rulePath)) {
+                $variables += $rule
```

Comment:
what happens if you have 2 paths for the same variable?  Would we set that variable twice?

Reply:
The `break;` below pops us out of the "path" forEach back into the "rule" forEach.  So we should only add a rule name once after we find the first matching path for the rule

Reply:
The rule name is the variable name, so we wouldn't have 2 rules with the same variable name